src/tests/functional/plugin/shared/src/single_layer_tests/is_inf.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
The test is placed in the wrong directory. Should be moved into shared_test_classes folder. 
Agreed to refactor in separate PR.